syntax = "proto3";

package ik_ChatServer;

message Request
{
    bytes type = 1;
    bytes request_msg = 2;
}

message AddGroupRequest
{
    int32 myid = 1;
    int32 group_id = 2;
}

message CreateGroupRequest
{
    int32 myid = 1;
    bytes group_name = 2;
}